supervisor/addons/validate.py

```diff
@@ -422,6 +423,7 @@ def _migrate(config: dict[str, Any]):
             vol.Coerce(int), vol.Range(min=10, max=300)
         ),
         vol.Optional(ATTR_JOURNALD, default=False): vol.Boolean(),
+        vol.Optional(ATTR_BREAKING_VERSIONS, default=list): [version_tag],
```

Comment:
I am not fully understand, why this is a list?

As in, you can only upgrade to a latest version, right? Would't that mean only the higher version number that is breaking would matter?

Reply:
User can still have an old version and this information prevent from auto update if they turn it on while people on never version run as it should.

Reply:
And the API support handling multiple versions, only git not. In theory we can use backups to provide ways to having multiple version. I think that way is flexible without having update the code + support a wide range of defragmentation

Reply:
Well tbf @pvizeli the feature is about auto-update. Auto-update is always going to go to the latest version. Manual update may not in the future but manual update won't check this list. We won't stop you from manually updating over a breaking change.

I guess I figured if a user was woefully out of date it could be useful to see all the versions with breaking changes they had to cross over. Also if the version scheme does change we might not be able to compare the addon's version with the latest versions but could still compare against old ones. It also just seemed like a useful record for users and the addon dev.

But I can change it to a simple string if we don't think its helpful.